cmd/abci-cli/abci-cli.go

```diff
@@ -228,9 +205,10 @@ func cmdEcho(c *cli.Context) error {
 	if len(args) != 1 {
 		return errors.New("Command echo takes 1 argument")
 	}
-	res := client.EchoSync(args[0])
-	rsp := newResponse(res, string(res.Data), false)
-	printResponse(c, rsp)
+	resEcho := client.EchoSync(args[0])
+	printResponse(c, response{
+		Data: resEcho.Data,
```

Comment:
why you wrap `resInfo.Data` into `[]byte` down below and do not wrap `resEcho.Data` here? small inconsistency caught my eye